src/content/docs/apm/agents/ruby-agent/getting-started/migration-9x-guide.mdx

```diff
@@ -226,28 +226,30 @@ See the [milestone for 9.0](https://github.com/newrelic/newrelic-ruby-agent/mile
+  Ruby 2.2 and 2.3 are no longer supported by the Ruby agent. To continue using the latest Ruby agent version, please update to Ruby 2.4.0 or higher.
 
-  Note that New Relic continues to provide support for certain older Ruby versions even after they have fallen out of the officially supported maintenance window provided by the Ruby language development team (Ruby team). For example, as of March 2023 the oldest Ruby version supported by the Ruby team will be version 3.0, whereas with version 9.0 of the New Relic agent, Ruby versions 2.4 and higher are still supported. For more information about official Ruby team support of Ruby versions, see [Ruby Maintenance Branches](https://www.ruby-lang.org/en/downloads/branches/).
+  New Relic continues to support certain older Ruby versions even after they have fallen out of the officially supported maintenance window provided by the Ruby language development team (Ruby team). For example, as of March 31, 2023 the oldest Ruby version supported by the Ruby team will be version 3.0. Whereas with version 9.0 of the New Relic agent, Ruby versions 2.4 and higher are still supported. For more information about official Ruby team support of Ruby versions, see [Ruby Maintenance Branches](https://www.ruby-lang.org/en/downloads/branches/).
 
 
 ## Enabled thread tracing by default [#thread_tracing]
 
-  The configuration option `instrumentation.thread.tracing` is now enabled by default. This configuration allows the agent to properly monitor code occurring inside threads. In Ruby agent 9.0, instrumented code within threads will be recorded and associated with the current transaction when the thread was created.
-This may be a breaking change if you're currently using custom thread instrumentation. New transactions inside of threads will no longer be started if one already exists.
+  The configuration option `instrumentation.thread.tracing` is now enabled by default. This configuration allows the agent to watch code occurring inside threads. Ruby agent 9.0 records instrumented code within threads and associates it with the current transaction. 
+  
+  This may be a breaking change if you're currently using custom thread instrumentation. New transactions inside of threads will no longer start if one already exists.
 
+  If you are using the old UI, you may see segments named "#" in your transaction charts. When looking at the segment table, the full name is "Thread: \#." These segments refer to thread instrumentation. See [Issue#1859](https://github.com/newrelic/newrelic-ruby-agent/issues/1859#issuecomment-1466774812) for details.
 
 ## Removed deprecated instrumentation [#removed_gem_instrumentation]
 
-  Instrumentation for the following gems had been previously deprecated and has now been removed.
+  Instrumentation for the following gems had been previously deprecated and is now removed.
```

Comment:
Check me on this one, but the combination of "is" and "removed" (a current state reference of a previously done thing) stands out. I like the change to use "is", but maybe now change "removed" to something else like "gone"?

```suggestion
  Instrumentation for the following gems had been previously deprecated and is now gone.
```

Reply:
Hmm... good question. I'm not sure. @ally-sassman, any advice here?

Reply:
You could say "The following instrumentation gems had been previously deprecated and are now no longer supported:"

Reply:
something about "gone" just feels weird to me. I found this discussion re: "is removed" - https://english.stackexchange.com/questions/31205/is-the-sentence-it-is-removed-grammatically-correct 

```suggestion
  Instrumentation for the following deprecated gems was removed.
```

Reply:
I think it's "was" instead of "were" since "instrumentation" is a singular subject.